notebooks/samples/201 - Amazon Book Reviews - TextFeaturizer.ipynb

```diff
@@ -137,8 +137,10 @@
    },
    "outputs": [],
    "source": [
-    "from mmlspark import FindBestModel\n",
-    "bestModel = FindBestModel(evaluationMetric=\"AUC\", models=lrmodels).fit(test)"
+    "from mmlspark import FindBestModel, BestModel\n",
```

Comment:
Please add similar tests to some of the other notebooks to validate that models produced by other estimators can be loaded as well.